### DISCUSSION TITLE
using scoppy with android WiFi hotspot.

### DISCUSSION THREAD

**Discussion by R4DH2N** [fhdm-dev/scoppy#106] [Q&A]
Hello, I have connected my pico W directly to my Android phones WiFi hotspot running scoppy on it.

Scoppy won't connect while WLAN is in Hotspot mode... The pico W connects fine to the hotspot.

Wy is scoppy relaying on WLAN client Status and doesn't recognize the enabled Hotspot.

I mean there is no need to depend on a specific network interface at all, as long as there is a network you can just use it and if not then access will fail.

**Comment by fhdm-dev** [comment]
Hi,
The Scoppy app doesn't specifiy a specific interface. It just uses whatever interface is the active one on the Android device so I'm not sure why it can't connect to the PicoW when the hotspot is enabled. I'll have to investigate. Thanks for letting me know about this.

**Comment by R4DH2N** [comment]
In case it matters, I have a Xiaomi Note 8t running Android 11 (MIUI 12.5.2)

I've also observed this behavior with other network-based apps, as soon as the WLAN is running as a hotspot, they can't seem to access it. It may have to do with Android permissions? I don't really know what's going on.

**Reply by fhdm-dev** [reply]
I think the problem is that when the hotspot is active, the network interface that Android considers the 'active' one is the cellular network. Scoppy always uses the active network but it should actually be checking all networks and using the Wi-Fi network interface in preference to the cellular network interface. Hopefully I'll have this fixed in the next version of the app.

**Reply by R4DH2N** [reply]
Thanks, I hope you can fix that.
